### DISCUSSION TITLE
Delete account?

### DISCUSSION THREAD

**Discussion by Neo-29** [stoatchat/discussions#434] [Q&A] [answered]
I'm tempted with this great alternative to discord.
I couldn't find this information.
If needed, can we delete account and server directly on line?
Tkd

**Comment by insertish** [comment, marked as answer]
Delete account in settings.

![image](https://user-images.githubusercontent.com/38285861/190851090-44a76279-1127-4a19-b73a-3b3813adc687.png)

**Reply by Neo-29** [reply]
Tks a lot for your quick reply.

**Reply by Rexogamer** [reply]
You can also delete a server in its settings:

![76C3B60E-DC63-4499-8B2F-8ACF5CF4FE37](https://user-images.githubusercontent.com/42586271/190857510-25fb1d3b-6dfe-47a7-88e5-1a90d53b3794.jpeg)
